### PR TITLE
MOTR-84 MOTR-62 MOTR-68 Frontend UI Enhancements

### PR DESCRIPTION
**Pull request by jimmyzhen**
**Key Changes:**
1. Created `motrpac-helpdesk@lists.stanford.edu` email address and updated **Contact** page with placeholder content.
2. Added config to remove `eslint` warnings on global browser variables (e.g. `window`, `document`).
3. Added Co-Investigators to the **Team** page and updated its UI styling.
4. Updated **Dashboard** page UI layout and its test assertions due to the UI changes.
5. Refactored **Landing** page to improve its responsiveness in various viewport sizes.
6. Updated **External Links** page UI and styling.
7. Minor UI and styling updates throughout other pages as well as navbar/sidebar for visual consistency.
8. Updated respective unit tests and **Storybook** stories due to various UI changes.

### CODE REVIEW

File: src/Navbar/navbar.jsx [hunk 1]
```diff
@@ -9,45 +9,44 @@ import MoTrPAClogo from '../assets/logo-motrpac.png';
...
 
   // Temp config to show/hide test interface visuals
   const urlParams = new URLSearchParams(window.location.search);
 
   // Function to render login button
   const LogoutButton = () => {
+    const userDisplayName = profile.user_metadata && profile.user_metadata.name
+      ? profile.user_metadata.name : profile.name;
+    const siteName = profile.user_metadata && profile.user_metadata.siteName
+      ? ', ' + profile.user_metadata.siteName : '';
```
Comment (kilodalton) [inline src/Navbar/navbar.jsx]: `eslint` is unhappy about this string concatenation

Comment (jimmyzhen) [reply src/Navbar/navbar.jsx]: Thank you for the great catch! I have addressed it in my latest commits.

Comment (kilodalton) [reply src/Navbar/navbar.jsx]: Thanks for converting it to a template.

**Comment by kilodalton** [comment]
Merged MOTR-84 MOTR-62 MOTR-68

**Pull request merged**